packages/date-time-picker/src/vaadin-date-time-picker.js

```diff
@@ -561,12 +592,10 @@ class DateTimePicker extends FieldMixin(SlotMixin(DisabledMixin(ThemableMixin(El
 
   /** @private */
   __i18nChanged(changeRecord) {
-    this.__datePickerChanged();
```

Comment:
Not 100% sure about this one, but I think it wasn't necessary. Changing i18n stuff updates the slotted element, and changing the slotted element (via `__datePickerChanged`) also updates the i18n [here](https://github.com/vaadin/web-components/pull/3153/files#diff-f4baf89fd06719c8fae9883b6b43b8dbd2e26daf73ddc6c9f492168b88e012c2R546) and [here](https://github.com/vaadin/web-components/pull/3153/files#diff-f4baf89fd06719c8fae9883b6b43b8dbd2e26daf73ddc6c9f492168b88e012c2R551).